lib/routes/nankai/ai-notice.ts

```diff
@@ -0,0 +1,142 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/ai/:type?',
+    categories: ['university'],
+    example: '/nankai/ai/zxdt',
+    parameters: { type: '栏目类型（若为空则默认为"最新动态"）' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['ai.nankai.edu.cn', 'ai.nankai.edu.cn/xwzx/:type.htm'],
+            target: '/ai/:type?',
+        },
+    ],
+    name: '人工智能学院',
+    maintainers: ['LMark'],
+    description: `| 最新动态 | 学院公告 | 学生之窗 | 科研信息 | 本科生教学 | 党团园地 | 研究生招生 | 研究生教学 | 就业信息 | 国际交流 |
+| -------- | -------- | -------- | -------- | ---------- | -------- | ---------- | ---------- | -------- | -------- |
+| zxdt     | xygg     | xszc     | kyxx     | bksjx      | dtyd     | yjszs      | yjsjx      | jyxx     | gjjl     |`,
+    url: 'ai.nankai.edu.cn',
+    handler: async (ctx) => {
+        // 从 URL 参数中获取通知分类
+        const { type = 'zxdt' } = ctx.req.param();
+        const baseUrl = 'https://ai.nankai.edu.cn';
+        const { data: response } = await got(`${baseUrl}/xwzx/${type}.htm`);
+        const $ = load(response);
+
+        // 获取分类名称映射
+        const categoryMap: Record<string, string> = {
+            zxdt: '最新动态',
+            xygg: '学院公告',
+            xszc: '学生之窗',
+            kyxx: '科研信息',
+            bksjx: '本科生教学',
+            dtyd: '党团园地',
+            yjszs: '研究生招生',
+            yjsjx: '研究生教学',
+            jyxx: '就业信息',
+            gjjl: '国际交流',
+        };
+
+        const categoryName = categoryMap[type] || '最新动态';
+
+        // 解析新闻列表
+        const list = $('table tr')
```

Comment:
This selects both tables from the left and right sides.
Update your selector to select table from `gage-list-news` class only.